utils/yoke/derive/src/lib.rs

```diff
@@ -13,13 +13,18 @@ use synstructure::Structure;
 
 /// Custom derive for `yoke::Yokeable`,
 ///
-/// If this fails to compile for lifetime issues, it means that
-/// the lifetime is not covariant and `Yokeable` is not safe to implement.
+/// If your struct contains `zerovec::ZeroMap`, then the compiler will not
+/// be able to guarantee the lifetime covariance due to the generic types on
+/// the `ZeroMap` itself. You must add the following attribute in order for
+/// the custom derive to work with `ZeroMap`.
 ///
-/// Note that right now this will fail to compile on structs involving
-/// `zerovec::ZeroMap`.
-/// Please comment on https://github.com/unicode-org/icu4x/issues/844
-/// if you need this
+/// ```text
```

Comment:
```suggestion
/// ```rust,ignore
```

Reply:
I want to push back on this one, but I could still be convinced otherwise.

With `text`, it shows up as just that:
![image](https://user-images.githubusercontent.com/58569820/133160007-d0f6ff99-5917-4cbd-b197-34d5d7cbb7c3.png)


But with `rust,ignore` it adds a big warning that this example is untested, and it's not really intended to be an example. 
![image](https://user-images.githubusercontent.com/58569820/133160056-655db96a-4ef1-46ad-a304-705c67498206.png)

Reply:
Yeah, it's still normal to use rust,ignore for that, but I guess it's not a big deal if this isn't syntax highlighted

Reply:
I'm happy either way. I'll just go ahead and push up the change.